airflow/example_dags/example_asset_decorator.py

```diff
@@ -18,9 +18,9 @@
 
 import pendulum
 
-from airflow.assets import Asset
 from airflow.decorators import dag, task
-from airflow.decorators.assets import asset
+from airflow.sdk.definitions.asset import Asset
+from airflow.sdk.definitions.decorators.asset import asset
```

Comment:
This is not just about `@asset` but I’ve always felt it’s somewhat awkward we put decorators in a different namespace. There’s not much wrong to just do

```python
from airflow.sdk.definitions.asset import asset
```

instead.

Reply:
cc @ashb

Reply:
yep, I feel the same way. Will move it to asset

Reply:
moved it to asset but split it in into different modules.

```tree
task_sdk/src/airflow/sdk/definitions/asset
├── __init__.py
├── decorators.py
└── metadata.py

```